### DISCUSSION TITLE
Why Isn't index: true Working to Show Files as Cards?

### DISCUSSION THREAD

**Discussion by kakarj** [fuma-nama/fumadocs#1109] [Q&A] [answered]
I’m trying to use `index: true` to display all the files in a folder as cards on the index page, but it’s not working on my docs. Is this custom or am I missing something?

it's working fine in fumadocs:
<img width="853" alt="Screenshot 2024-11-24 at 9 37 13 PM" src="https://github.com/user-attachments/assets/f0e03589-f1c5-40bf-b839-444838b114bd">
<img width="853" alt="Screenshot 2024-11-24 at 9 36 54 PM" src="https://github.com/user-attachments/assets/bbd31853-0314-489f-93d2-c36bdf3b2a3e">

**Comment by fuma-nama** [comment, marked as answer]
Use https://fumadocs.vercel.app/docs/ui/blocks/page#category

**Reply by kakarj** [reply]
I tried using this code:  

```jsx  
{page.data.index ? <DocsCategory page={page} from={source} /> : null}  
```  

from the Fumadocs repo, but it’s throwing the following error:  

```
Property 'index' does not exist on type 'Omit<MarkdownProps, "title" | "icon" | "description" | "full" | "_openapi"> & { title: string; icon?: string | undefined; description?: string | undefined; full?: boolean | undefined; _openapi?: objectOutputType<...> | undefined; } & BaseCollectionEntry'.
```  

It seems like `index` isn’t specified. How do I fix this and use this?

**Reply by fuma-nama** [reply]
It's a custom frontmatter data, see https://fumadocs.vercel.app/docs/mdx/collections#schema-1 to extend the schema
